src/style.css

```diff
@@ -0,0 +1,95 @@
+:root {
+  --padding-left-big-search: 62px;
+  --padding-left-small-search: 33px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.forms-wrapper {
+  margin-top: 20px;
+}
+
+.label-search-icon {
```

Comment:
This could be done like so
```
position: absolute;
top: calc(var(--big-search-height) / 2); -> height of the big from could be a variable
left: calc(var(--padding-left-big-search) / 2);
transform: translateY(-50%) translateX(-50%);
cursor: text;
```

Reply:
Similarly for the small label